Group VS Code topics together

rjamesak opened the pull request:
VS Code information has been moved so that the sections can be found together in the documentation.

deltakosh commented:
Do we need to run the script again or will it update when I will push on prod?

QuentinRillet commented:
Yes, it is necessary to restart the script for the compilation of the documentation. I only committed the script modification.

deltakosh commented:
Do you mind generating it?

QuentinRillet commented:
Yes, I do it quickly

QuentinRillet commented:
In fact it is automatically generated on the server at the time of deployment. You can merge on the prod branch :) @deltakosh

rjamesak committed:
Group VS Code topics together

VS Code information has been moved so that the sections can be found together in the documentation.
